### ISSUE TITLE
Odd but I can see the damn'ed cat just fine. But I'm missing a src attribute in the <img> element I'm being told?

### ISSUE THREAD

**Issue by glennaxworthy** [FreeCodeCamp/FreeCodeCamp#9437]
Challenge [Add Images to your Website](https://www.freecodecamp.com/challenges/add-images-to-your-website#?solution=%0A%3Clink%20href%3D%22https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DLobster%22%20rel%3D%22stylesheet%22%20type%3D%22text%2Fcss%22%3E%0A%0A%3Cstyle%3E%0A%20%20.red-text%20%7B%0A%20%20%20%20color%3A%20red%3B%0A%20%20%7D%0A%0A%20%20h2%20%7B%0A%20%20%20%20font-family%3A%20Lobster%2C%20Monospace%3B%0A%20%20%7D%0A%0A%20%20p%20%7B%0A%20%20%20%20font-size%3A%2016px%3B%0A%20%20%20%20font-family%3A%20Monospace%3B%0A%20%20%7D%0A%3C%2Fstyle%3E%0A%0A%3Ch2%20class%3D%22red-text%22%3ECatPhotoApp%3C%2Fh2%3E%0A%3Cimg%20src%3D%22https%3A%2Fbit.ly%2Ffcc-relaxing-cat%22%3E%0A%0A%3Cp%20class%3D%22red-text%22%3EKitty%20ipsum%20dolor%20sit%20amet%2C%20shed%20everywhere%20shed%20everywhere%20stretching%20attack%20your%20ankles%20chase%20the%20red%20dot%2C%20hairball%20run%20catnip%20eat%20the%20grass%20sniff.%3C%2Fp%3E%0A%3Cp%20class%3D%22red-text%22%3EPurr%20jump%20eat%20the%20grass%20rip%20the%20couch%20scratched%20sunbathe%2C%20shed%20everywhere%20rip%20the%20couch%20sleep%20in%20the%20sink%20fluffy%20fur%20catnip%20scratched.%3C%2Fp%3E) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/601.6.17 (KHTML, like Gecko) Version/9.1.1 Safari/601.6.17</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">

<style>
  .red-text {
    color: red;
  }

  h2 {
    font-family: Lobster, Monospace;
  }

  p {
    font-size: 16px;
    font-family: Monospace;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>
<img src="https:/bit.ly/fcc-relaxing-cat">

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
<p class="red-text">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>
```

**Comment by dwd2010** [comment]
@glennaxworthy, 
Thanks for reporting this. But your code is incorrect. 
You forget to add / in src.  you wrote `<img src="https:/bit.ly/fcc-relaxing-cat">`. 
correct answer is `<img src="https://bit.ly/fcc-relaxing-cat">`
GitHub Issues are for reporting bugs on the website only.
Please use the [Help chat room](https://gitter.im/FreeCodeCamp/Help) for getting challenge related help.

Happy Coding!

**Comment by dwd2010** [comment]
I meant to say you forget to add / after https:
you put one of /. it suppose to have two /.

**Comment by glennaxworthy** [comment]
You are correct. The fact that it worked just fine on the sample output obscured the issue. Better that it not work so I have a clue where to look.
I AM impressed by the rapid response!
Thanks

**Comment by glennaxworthy** [comment]
yes, I got that. It's a great shame that "https:/..." worked just fine on your simulator so who would guess that there was a typo in the url? I type the stuff in myself rather than cut and paste in hopes that doing so drives the point in. But if the image hadn't appeared, the first thing I would have looked at would be the url itself. Basic debugging. Instead I was told I hadn't specified the url. That was incomplete and confusing information since I had indeed supplied one that worked. I do believe this to be a bug on your end with regards to the support of incorrect urls and reportage of the problem. Me? I made a typo.
